### PR TITLE
ROX#18584: Add info for changing collection method to upgrade instructions

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
- merge to `rhacs-docs`
- cherrypick to 4.4 (see separate PR for 4.3)

[Issue](https://issues.redhat.com/browse/ROX-18584)

[Link to docs preview
](https://72950--docspreview.netlify.app/openshift-acs/latest/upgrading/upgrade-operator#prepare-operator-upgrades_upgrade-operator)

QE review:
- [x] QE has approved this change.

Additional information:

See #72968 for 4.3

### CODE REVIEW

File: modules/change-collection-method-ebpf.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+If the cluster that you are upgrading contains the `SecuredCluster` CR, you must ensure that the per node collection setting is set to EBPF before you upgrade.
+
+.Procedure
+
+. In the {ocp} web console, go to the {product-title-short} Operator page.
+. In the top navigation menu, select *Secured Cluster*.
+. Click the instance name, for example, *stackrox-secured-cluster-services*.
+. Click YAML to open the YAML editor.
+. Locate the `spec.perNode.collector.collection` attribute.
+. If the value is `KernelModule`, then change it to `EBPF`.
```
Comment (JoukoVirtanen) [inline modules/change-collection-method-ebpf.adoc:18]: For 4.4 they should upgrade it to CORE_BPF

File: modules/prepare-operator-upgrades.adoc [hunk 2]
```diff
@@ -10,3 +10,4 @@ Before you upgrade {rh-rhacs-first} version, you must:
 
 * Verify that you are running the latest patch release version of the {product-title-short} Operator 3.74.
 * Backup your existing Central database.
+* Uf the cluster you are upgrading contains the `SecuredCluster` custom resource (CR), change the collection method to `CORE_BPF`.
```
Comment (tmalove) [inline modules/prepare-operator-upgrades.adoc:13]:
```suggestion
* If the cluster you are upgrading contains the `SecuredCluster` custom resource (CR), change the collection method to `CORE_BPF`.
```

**Review by tmalove — APPROVED** [review]

**Pull request merged**